packages/admin-portal/src/components/pages/approvals/approvals.tsx

```diff
@@ -161,7 +161,7 @@ export const generateTableColumn = ({
 export const AdminApprovals: React.FC = () => {
   const [isModalOpen, setIsModalOpen] = React.useState(false)
   const dispatch = useDispatch()
-  const appDetail = useSelector(selectApp)
+  const appDetail = useSelector(selectAppDetailData) as AppDetailModel
```

Comment:
Why do we need to cast it to `AppDetailModel` here? Its type should be defined in `selectAppDetailData` function